core/src/main/java/org/testcontainers/containers/ContainerState.java

```diff
@@ -71,9 +72,12 @@ default boolean isHealthy() {
 
         try {
             InspectContainerResponse inspectContainerResponse = getCurrentContainerInfo();
-            String healthStatus = inspectContainerResponse.getState().getHealth().getStatus();
+            HealthState health = inspectContainerResponse.getState().getHealth();
+            if (health == null) {
+                throw new RuntimeException("Healthchecks not configured for this container");
```

Comment:
I think it would be good to give the user a little more of a hint about what has gone wrong/what they can do.

Perhaps:
```suggestion
                throw new RuntimeException("This container's image does not have a healthcheck declared, so health cannot be determined. Either amend the image or use another approach to determine whether containers are healthy.");
```

Reply:
🥇